docs/cloud-guidance/azure/ukho-azure-tagging.md

```diff
@@ -0,0 +1,28 @@
+# UKHO Azure Tagging Strategy v0.1 DRAFT
+
+## How to use this document
+
+This document outlines which tags should be applied when creating resources in the cloud. Use the table below to ensure you are conforming to the UKHO tagging strategy.
+
+## How to contribute to this document
+
+The owner of this document is Digital Operations. Please submit suggestions/changes/amendments vie a pull request and inform Digital Operations.
+
+## References
```

Comment:
Move this above `How to contribute to this document`

Reply:
Done